src/Servers/Kestrel/Core/test/AsciiDecoding.cs

```diff
@@ -147,7 +147,7 @@ static void Test(Span<byte> asciiBytes)
 
                     // Change byte back for next iteration, ensure is equal again
                     asciiBytes[i] = b;
-                    Assert.True(StringUtilities.BytesOrdinalEqualsStringAndAscii(s, asciiBytes));
+                    Assert.True(StringUtilities.BytesOrdinalEqualsStringAndAscii(s, asciiBytes), s);
```

Comment:
While I'm here, might as well add some additional context for when this might fail